src/plugins/core.js

```diff
@@ -394,10 +394,28 @@ function Plugin(options = {}) {
     debug('onDropText', { data })
 
     const { text, target } = data
+    const { document } = state
     const transform = state
       .transform()
       .select(target)
 
+    let hasVoidParent = document.hasVoidParent(target.get('anchorKey'))
+
+    // For void targets, we want to find the nearest non-void text node
+    if (hasVoidParent) {
+      let node = document.getNode(target.get('anchorKey'))
```

Comment:
Can you change all of these `.get(` calls to just use property notation, like `target.anchorKey` instead? Thanks!